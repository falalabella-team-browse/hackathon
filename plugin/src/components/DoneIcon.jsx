import React from "react";

const DoneIcon = () => {
  return (
    <svg
      id="Layer_1"
      viewBox="0 0 511.92 511.92"
      width="40px"
      height="40px"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="m501.376 92.099-31.054-30.059c-13.687-13.249-35.956-13.249-49.644 0l-260.44 252.109c-1.366 1.323-3.764 1.322-5.131 0l-64.268-62.21c-13.687-13.249-35.956-13.249-49.643 0l-30.652 29.67c-6.799 6.581-10.544 15.371-10.544 24.751s3.745 18.17 10.544 24.751l92.714 89.746c.012.011.023.022.035.033l29.951 28.992c6.844 6.625 15.831 9.937 24.821 9.936 8.988 0 17.979-3.313 24.822-9.936l318.489-308.281c6.799-6.581 10.544-15.371 10.544-24.75s-3.744-18.17-10.544-24.752z"
        fill="#aad500"
      />
    </svg>
  );
};

export default DoneIcon;
