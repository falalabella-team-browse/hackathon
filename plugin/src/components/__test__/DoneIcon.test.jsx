import React from "react";
import { shallow } from "enzyme";
import DoneIcon from "../DoneIcon";

describe("DoneIcon", () => {
  it("should render properly", () => {
    const wrapper = shallow(<DoneIcon />);
    expect(wrapper).toMatchSnapshot();
  });
});
