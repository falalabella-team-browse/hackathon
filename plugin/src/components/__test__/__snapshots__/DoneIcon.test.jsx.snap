// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DoneIcon should render properly 1`] = `ShallowWrapper {}`;
